src/sdk/pynni/nni/metis_tuner/metis_tuner.py

```diff
@@ -67,40 +65,45 @@ class MetisTuner(Tuner):
     """
 
     def __init__(self, optimize_mode="maximize", no_resampling=True, no_candidates=True,
-                 selection_num_starting_points=10, cold_start_num=10):
+                 selection_num_starting_points=600, cold_start_num=10, exploration_probability=0.1):
```

Comment:
why adjust those default parameters? any changes introduced regarding the tuner's default behaviour?

Reply:
we adjust this parameter due to the experiment result.